src/manage.c

```diff
@@ -284,6 +283,22 @@ u8 manage_track_new_acq(float snr)
   return MANAGE_NO_CHANNELS_FREE;
 }
 
+static void check_clear_unhealthy(void)
+{
+  static u16 wn;
+  gps_time_t t = get_current_time();
```

Comment:
Consider using `chTimeNow` instead as `get_current_time` reads from the FPGA

http://chibios.sourceforge.net/html/group__time.html